src/Services/EncounterService.php

```diff
@@ -17,6 +17,8 @@
 namespace OpenEMR\Services;
 
 use Particle\Validator\Validator;
+use OpenEMR\Common\Acl\AclMain;
+use OpenEMR\Services\FacilityService;
```

Comment:
lets keep the `use` stuff sorted alphabetically

Reply:
done